src/zac/advices/templates/advices/includes/approval_table.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<table class="table">
+    <thead>
+        <tr>
+            <th class="table__header">{% blocktrans %}Approval{% endblocktrans %}</th>
+            <th class="table__header">{% blocktrans %}From{% endblocktrans %}</th>
+            <th class="table__header">{% blocktrans %}Given on{% endblocktrans %}</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        {% for approval in approvals %}
+            <tr>
+                <td>
+                    {% if approval.approved %}
+                        {% blocktrans %}Approved{% endblocktrans %}
+                    {% else %}
+                        {% blocktrans %}Not approved{% endblocktrans %}
+                    {% endif %}
+                </td>
+                <td> {% firstof approval.author.get_full_name approval.author.username %}</td>
+                <td>
+                    <time title="{{ approval.created.isoformat }}">
+                        {{ approval.created|timesince }} {% blocktrans %}ago{% endblocktrans %}
+                    </time>
```

Comment:
Please review the documentation for [`blocktrans`](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#std:templatetag-blocktrans) - 

```suggestion
                    <time title="{{ approval.created.isoformat }}">
                        {% blocktrans with created_since=approval.created|timesince trimmed %}
                            {{ created_since }} ago
                        {% endblocktrans %}
                    </time>
```

The importance here is that the order of `5 days ago` is not the same in every language - other language could have `foo {{ created_since }}` in the translated form. That's why you need to put the entire construct in the `{% blocktrans %}` tags.